configs/maheswaranathan_core_readout.yaml

```
@@ -27,7 +27,7 @@ trainer:
 
 dataloader:
   batch_size: null
-  num_workers: 4
+  num_workers: 3
```

directly set to 0 then?
```suggestion
  num_workers: 0
```

---

I will add it in a new PR with temporary notimplementederrors to warn the user if they try to increase it